chrome/content/zotero/integration/integrationDocPrefs.xul

```diff
@@ -77,4 +84,4 @@
 		<checkbox id="storeReferences" label="&zotero.integration.prefs.storeReferences.label;"/>
 		<description class="radioDescription">&zotero.integration.prefs.storeReferences.caption;</description>
 	</vbox>
-</dialog>
\ No newline at end of file
+</dialog>
```

Comment:
Yeah, that sounds complicated. I can just add the reversion to @aurimasv commit, instead of going back in time and erasing the change.